clients/unshaded/src/main/java/tachyon/client/file/FileSystemContext.java

```diff
@@ -15,14 +15,23 @@
 
 package tachyon.client.file;
 
+import javax.annotation.concurrent.ThreadSafe;
+
 import tachyon.client.ClientContext;
 import tachyon.client.block.TachyonBlockStore;
 
 /**
  * A shared context in each client JVM for common file master client functionality such as a pool of
- * master clients. Any remote clients will be created and destroyed on a per use basis. This class
- * is thread safe.
+ * master clients. Any remote clients will be created and destroyed on a per use basis.
+ *
+ *
```

Comment:
I am not sure if empty line is always respected in javadoc (at least not in Intellij), while `<p>` forces a new paragraph.